lib/open_meteo/response_wrapper.rb

```diff
@@ -0,0 +1,40 @@
+module OpenMeteo
+  ##
+  # Wrap the JSON body response from the OpenMeteo request.
+  #
+  class ResponseWrapper
+    def initialize(config: OpenMeteo::Client::Config.new)
+      @config = config
+    end
+
+    def wrap(response, entity: OpenMeteo::Entities::Forecast)
```

Comment:
```suggestion
    def wrap(response, entity:)
```

My previous comment was also about removing this here.